expyfun/analyze/_analyze.py

```diff
@@ -254,7 +295,8 @@ def rt_chisq(x, axis=None):
     if axis is None:
         df, _, scale = ss.chi2.fit(x, floc=0)
     else:
-        fit = partial(ss.chi2.fit, floc=0)
+        def fit(x):
+            return np.array(ss.chi2.fit(x, floc=0))
```

Comment:
It won't work with newest `numpy` that requires that whatever is passed to `apply_along_axis` give a `np.ndarray` out instead of, in this case, a tuple